docs/pages/includes/edition-prereqs-tabs.mdx

```diff
@@ -36,7 +54,7 @@
 
 </TabItem>
 <TabItem scope={["cloud"]}
-  label="Teleport Cloud">
+  label="Teleport Enterprise Cloud">
```

Comment:
@ptgott Should this remain just "Cloud" for instructions that apply to both Team and Enterprise Cloud, and have the commercial prereqs partial for when only Enterprise Cloud applies?

Reply:
When I created the original PR, I assumed that Team wouldn't require the Enterprise `tctl` client, so I made a separate tab. If Team does require Enterprise `tctl`, though, I think we can use "Teleport Team/Enterprise Cloud" as the label and use the same tab item for both Team and Enterprise Cloud.

Reply:
I've confirmed that teams users should use enterprise `tctl`.

Reply:
I've backported https://github.com/gravitational/teleport/pull/29024 into this PR